src/funding/applepay/config.jsx

```diff
@@ -0,0 +1,48 @@
+/* @flow */
+/** @jsx node */
+
+import { PLATFORM } from '@paypal/sdk-constants/src';
+import { ApplePayLogo, LOGO_COLOR } from '@paypal/sdk-logos/src';
+
+import { BUTTON_COLOR, BUTTON_LAYOUT } from '../../constants';
+import { DEFAULT_FUNDING_CONFIG, type FundingSourceConfig } from '../common';
+
+export function getApplePayConfig() : FundingSourceConfig {
+    return {
+        ...DEFAULT_FUNDING_CONFIG,
+
+        requiresPopupSupport:             false,
+        requiresSupportedNativeBrowser:   false,
+        shippingChange:                   true,
+
+        platforms: [
+            PLATFORM.DESKTOP,
+            PLATFORM.MOBILE
+        ],
+
+        layouts: [
+            BUTTON_LAYOUT.HORIZONTAL,
+            BUTTON_LAYOUT.VERTICAL
+        ],
+
+        Logo: ({ logoColor, optional }) => ApplePayLogo({ logoColor, optional }),
+
+        colors: [
+            BUTTON_COLOR.BLACK,
+            BUTTON_COLOR.WHITE
+        ],
+
+        logoColors:  {
+            [ BUTTON_COLOR.BLACK ]:  LOGO_COLOR.WHITE,
+            [ BUTTON_COLOR.WHITE ]:  LOGO_COLOR.BLACK
+        },
+
+        eligible: () => {
+            if (window.ApplePaySession && window.ApplePaySession.canMakePayments()) {
```

Comment:
Can't do this here either :) gotta pass it as a prop. Will be no `window` on the server.

Reply:
@bluepnume  Shoot! Right!  I keep looking at this as being on the client. Wondering about creating another product specific prop though so we don't have to keep creating these for each new button.  Maybe just an 'isSupported' for all?